system-tests/test/deposit-redemption.test.ts

```diff
@@ -410,34 +470,54 @@ describe("System Test - Deposit and redemption", () => {
   })
 
   context("when deposit is made and revealed with a vault", () => {
```

Comment:
I think this scenario is redundant. We are doing exactly the same in `minting-unminting.test.ts`. We can safely remove it and rename the earlier scenario `when deposit is made and revealed without a vault` back to `when deposit is made and revealed`.

Reply:
Removed in 12e215f04cab22674375a0d2763f6a1337a1a284.